include/SFML/Config.hpp.in

```diff
@@ -35,10 +35,25 @@
 ////////////////////////////////////////////////////////////
 // Define the SFML version
 ////////////////////////////////////////////////////////////
-#define SFML_VERSION_MAJOR      3
-#define SFML_VERSION_MINOR      0
-#define SFML_VERSION_PATCH      0
-#define SFML_VERSION_IS_RELEASE false
+// clang-format off
+#define SFML_VERSION_MAJOR @SFML_VERSION_MAJOR@
+#define SFML_VERSION_MINOR @SFML_VERSION_MINOR@
+#define SFML_VERSION_PATCH @SFML_VERSION_PATCH@
+// clang-format on
```

Comment:
Why disable clang-format?

Reply:
`clang-format` puts a space before the last at sign, so I just disable it here. I should probably add .in as well to formatting to keep this file formatted like before.